src/telegram/ext/_application.py

```diff
@@ -780,8 +781,11 @@ def run_polling(
         Args:
             poll_interval (:obj:`float`, optional): Time to wait between polling updates from
                 Telegram in seconds. Default is ``0.0``.
-            timeout (:obj:`int`, optional): Passed to
-                :paramref:`telegram.Bot.get_updates.timeout`. Default is ``10`` seconds.
+            timeout (:obj:`int` | :class:`datetime.timedelta`, optional): Passed to
+                :paramref:`telegram.Bot.get_updates.timeout`. Default is ``timedelta(seconds=10)``.
```

Comment:
I forgot the exact sphinx syntax, but we can link to the timedelta docs while telling that it's 10 seconds, it might be something like this (we've used it in other places in the lib):

```suggestion
                :paramref:`telegram.Bot.get_updates.timeout`. Default is :obj:`timedelta(seconds=10)<datetime.timedelta>`.
```